applications/dashboard/openapi/search.yml

```diff
@@ -32,59 +39,52 @@ paths:
           type: array
         style: form
       - description: >
-          <p>Set the scope of the search to the comments of a discussion.
-          Incompatible with recordType and type.</p>
+          Set the scope of the search to the comments of a discussion.
+          Incompatible with recordType and type.
         in: query
         name: discussionID
         schema:
           type: integer
         x-search-scope: true
       - description: |
-          <p>Set the scope of the search to a specific category.</p>
+          Set the scope of the search to a specific category.
         in: query
         name: categoryID
         schema:
           type: integer
         x-search-scope: true
       - description: |
-          <p>Set the scope of the search to followed categories only.</p>
+          Set the scope of the search to followed categories only.
         in: query
         name: followedCategories
         schema:
           default: false
           type: boolean
         x-search-scope: true
       - description: >
-          <p>Search the specified category's subtree. Works with
-          categoryID</p>
+          Search the specified category's subtree. Works with
+          categoryID
         in: query
         name: includeChildCategories
         schema:
           default: false
           type: boolean
       - description: |
-          <p>Allow search in archived categories.</p>
+          Allow search in archived categories.
         in: query
         name: includeArchivedCategories
         schema:
           default: false
           type: boolean
       - description: |
-          <p>Filter the records using the supplied terms.</p>
-        in: query
-        name: query
-        schema:
-          type: string
-        x-search-filter: true
-      - description: |
-          <p>Filter the records by matching part of their name.</p>
```

Comment:
I believe that only last line supposed to be replaced. All the rest get to this PR by accident I guess.
Because param with: `name: query` is one of our key param for  advanced search api:
https://github.com/vanilla/internal/blob/4671a163429f705d18de89b72c881abc1fed3c48/plugins/AdvancedSearch/controllers/api/SearchApiController.php#L153